doc/en/components/grids/_shared/cascading-combos.md

```diff
@@ -30,7 +30,7 @@ In order enable column editing, make sure `editable` property is set to `true`.
 Once the column editing is enabled, you can start by adding your `Combo`. Please note that here in order to have only one single selection available, you will need to use set the `singleSelect` property.
 
 
-<!-- WebComponents, Blazor -->
+<!-- WebComponents, Blazor, React -->
 To get started with the `Combo`, first you need to import it:
 
 ```ts
```

Comment:
This is not relevant to react

Reply:
Why not? The section can be enabled to explain the content of the sample that already exists for react.